charts/common/templates/_service.yaml

```
@@ -0,0 +1,34 @@
+{{/* 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)